libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFSmartStoreTests.m

```diff
@@ -690,7 +690,7 @@ - (void)removeStoreForManager:(SFSmartStoreDatabaseManager *)dbMgr withName:(NSS
 - (void)createDbDir:(NSString *)dbName withManager:(SFSmartStoreDatabaseManager *)dbMgr
 {
     NSError *createError = nil;
-    [dbMgr createStoreDir:dbName error:&createError];
+    [dbMgr createStoreDir:dbName];
     XCTAssertNil(createError, @"Error creating store dir: %@", [createError localizedDescription]);
```

Comment:
You are right, we should XCTAssertTrue the return value of [dbMgr createStoreDir]